infertrade/algos/community/signals.py

```diff
@@ -130,11 +130,4 @@ def scikit_signal_factory(signal_function: callable):
             "github_permalink": "https://github.com/ta-oliver/infertrade/blob/df1f6f058b38e0ff9ab1250bb43ffb220b3a4725/infertrade/algos/community/signals.py#L55"
         },
     },
-}
-
-
-def test_NormalisedCloseTransformer():
-    """nct = NormalisedCloseTransformer()"""
-    nct = scikit_signal_factory(normalised_close)
-    X = nct.fit_transform(simulated_market_data_4_years_gen)
-    assert isinstance(X, np.array)
```

Comment:
@bi-kash - move this to the tests folder? Or we have an alternative test for scikit_signal_factory ?

Reply:
Generally all tests should be within `/tests`

Reply:
It's already handled in test_examples.py.

Reply:
It's already handled in test_examples.py.